python/ray/train/backends/__init__.py

```diff
@@ -1,7 +1,9 @@
 from ray.train.backends.backend import BackendConfig
 from ray.train.backends.horovod import HorovodConfig
 from ray.train.backends.tensorflow import TensorflowConfig
-from ray.train.backends.torch import TorchConfig
+
+# Backwards Compatiblity
+from ray.train.torch import TorchConfig
```

Comment:
Is this change needed? Doesn't `ray.train.torch` just point back to `ray.train.backends.torch`?

Reply:
Can you also explain what the long term plan for this is? Maybe remove `backends` package entirely?

Reply:
The idea is that from a user-facing API perspective, it's better to do `model = train.torch.prepare(model)` instead of `model = train.backends.torch.prepare(model)`.

I think ideally yes, we can remove the `backends` package entirely.

Reply:
Oh yeah I definitely agree with that, the original comment was just about how `TorchConfig` is defined in `python/ray/train/backends/torch.py` so this import is `ray.train.backend.torch` -> `ray.train.torch.` -> `ray.train.backend.torch`.

Reply:
Did the refactoring in a separate PR! https://github.com/ray-project/ray/pull/20312